### ISSUE TITLE
PodSecurityPolicy is deprecated in Kubernetes v1.25

### ISSUE THREAD

**Issue by budimanjojo** [alex1989hu/kubelet-serving-cert-approver#106]
Hello, thanks for this awesome tool. [PSP is now deprecated with v1.25](https://kubernetes.io/blog/2021/04/08/kubernetes-1-21-release-announcement/#podsecuritypolicy-deprecation). Is there any plan of supporting this new behavior? I just upgraded to `v1.25` and although everything is currently running just fine, but the apiserver keeps complaining about this:
`PodSecurityPolicy/kubelet-serving-cert-approver namespace not specified, error: the server could not find the requested resource`.

Thanks before.

**Comment by alex1989hu** [comment]
Dear @budimanjojo thank you for your feedback ☺️ I have just created a [new release](https://github.com/alex1989hu/kubelet-serving-cert-approver/releases/tag/v0.6.0) which [removes PSP usage and it applies strict Pod Security Standards](https://github.com/alex1989hu/kubelet-serving-cert-approver/commit/e41795fb0cebb5aed2a20bc3e53294c485524505#diff-0db16276644fc4f21eae05d761a230f6438feca8b7c0a92427c9d3b3ea85726a).

If the `PodSecurity` feature gate is not enabled in your cluster then it will not have any affect.

Let me paste section from **deploy/README.md**:

```md
* Configured Pod Security Standards
  * Applied Pod Security Admission labels onto the namespace with `restristed` Pod Security Standards profile
  * Only active when `PodSecurity` feature gate is enabled in your cluster
```

The images with `0.6.0` tag will be published as soon as the pipeline finishes.

PS: to honest I don't know what I did when I created `v0.5.1` tag: we see commits before `v0.5.1`  even if we compare `v0.5.1` ... `v0.6.0` tags _(actually we see all commits after `v0.5.0`)_.
